CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGES:
 * Vault default image to 1.9.3
 * CSI provider default image to 1.1.0
 * Earliest Kubernetes version tested is now 1.16
+* Support topologySpreadConstraints in server and injector. [GH-614](https://github.com/hashicorp/vault-helm/pull/614) and [GH-652](https://github.com/hashicorp/vault-helm/pull/652)
```

Comment:
Since it looks like 652 contains all the commits from 614, I'd say just mention 652 here, since the other wasn't merged.

```suggestion
* Support topologySpreadConstraints in server and injector. [GH-652](https://github.com/hashicorp/vault-helm/pull/652)
```